discord/ext/test/factories.py

```diff
@@ -186,6 +186,8 @@ def dict_from_channel(channel):
 def make_message_dict(channel, author, id_num=-1, content=None, timestamp=None, edited_timestamp=None, tts=False,
                       mention_everyone=False, mentions=None, mention_roles=None, mention_channels=None,
                       attachments=None, embeds=None, pinned=False, type=0, **kwargs):
+    if not content:
```

Comment:
Can instead do `content=""` in the function params

Reply:
So if you do that and somehow None get's passed to content, then it will break again.  I'm not actually sure if that's possible, but if it is... It could be a problem.  I'll submit the way it is and if you choose to merge you can then decide to change?